lib/app/models/inferno_test.rb

```diff
@@ -0,0 +1,64 @@
+module Inferno
+  module Sequence
+    class InfernoTest
```

Comment:
Opinions on a better name?

Reply:
Does SequenceTest sound better?

Reply:
I think it seems about the same. Maybe it should just be `Test`? `Inferno::Sequence::Test`

Reply:
My only objection to that is that Test is fairly generic and used by other libraries.  But that is weak.  Sequence::Test seems nice to me.  My real problem is that what we define as a 'Test' may defy expectations a bit because they behave differently based on context.   But that can't really be solved here.

Reply:
TestStep?

Reply:
what do MiniTest/Rspec call them?

Reply:
I have been noodling with calling each of these 'steps' instead of 'tests'.  But that would be a pretty big change and perhaps this is something we can just keep in the back of our minds. Let's keep this as an 'InfernoTest' for now...